backend/composer/models.py

```diff
@@ -713,6 +712,20 @@ class Meta:
         ]
 
 
+class GraphRenderingState(models.Model):
+    connectivity_statement = models.OneToOneField(
+        ConnectivityStatement,
+        on_delete=models.CASCADE,
+        related_name='graph_rendering_state',
+    )
+    serialized_graph = models.JSONField()  # Stores the serialized diagram model
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+    saved_by = models.ForeignKey(
```

Comment:
@afonsobspinto question, are the update_at and saved_by used to update the change in the notes? Thanks

Reply:
No. At the moment they are not used but I added them anyway because I can see it being useful for something like that.

Reply:
We can also have a note in the graph saying. Last saved by: {saved_by} at: {updated_at}